src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -11,20 +11,20 @@ function renderPaginationLink(data, index){
 
 	// RENDER
 	// --------------------------
-	const { number } = data;
+	const { episodeNo } = data;
```

Comment:
don't forget to update this once the data-object has been udpated!

Reply:
Created episode-nav-state branch off episode-preview to do this. Will merge down once done.

Reply:
All working and merged back to episode-preview.